pkg/core/util.go

```diff
@@ -25,6 +25,36 @@ func CreateGenesisBlock(cfg config.ProtocolConfiguration) (*block.Block, error)
 		return nil, err
 	}
 
+	txs := []*transaction.Transaction{}
+	for i, s := range cfg.GenesisExtensions.Transactions {
+		signers := make([]transaction.Signer, len(s.Signers)+1)
+		signers[0] = transaction.Signer{
+			Account: nextConsensus,
+			Scopes:  transaction.None, // Enough to pay for the transaction.
+		}
+		for j := range signers[1:] {
```

Comment:
One potential problem I see with this approach is that if we're going to deploy NeoFS contracts with the help of this extension, then the transaction sender is always validator multisig, and contract's hash will always depend on it. Not sure, is it good or bad for our contracts.

Reply:
They don't care much.